### ISSUE TITLE
Cannot deal with Angular labels

### ISSUE THREAD

**Issue by Danielmethner** [microsoft/vscode-edge-devtools#1725]
**Environment (please complete the following information):**
 - Windows 11
 - Extension version 2.1.3

**Describe the bug:**
Angular allows the adding of dynamic aria-labels like in this example:
<button [attr.aria-label]="myActionLabel">…</button>
This is described in the documentation here: https://angular.io/guide/accessibility

Expected Behaviour:
The extension recognises this and the warning "Form elements must have labels: Element has no title attribute Element has no placeholder attributeMicrosoft Edge Tools[axe/forms](https://dequeuniversity.com/rules/axe/4.8/label?application%3DaxeAPI)" is no longer shown.

Actual behaviour:
The extension still acts as if the attribute was missing.

**Repro steps:**
1. Add an element with a dynamic aria label to your application:
    <input type="number" [attr.aria-label]="myInput.ariaLabel" />

2. Activate the plugin.

The input field will be marked as missing an aria-label.

**Expected behavior:**
The plugin should recognise the attribute [attr.aria-label] and no longer show the warning.

**Additional context:**

**Comment by captainbrosset** [comment]
Thank you, this makes sense. Let me make sure this shows up on the team's list.
